packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -43,7 +43,7 @@ jest.mock(`gatsby-core-utils`, () => {
   return {
     slash: originalCoreUtils.slash,
     cpuCoreCount: jest.fn(() => `1`),
-    createContentDigest: jest.fn(() => `contentDigest`),
+    createContentDigest: originalCoreUtils.createContentDigest,
```

Comment:
Seems better to me to use the real `createContentDigest` function. Although the snapshots look more random now, it's still [a deterministic value](https://github.com/gatsbyjs/gatsby/blob/58e28ac14b9247f335651cfc2125396ff6d7a13c/packages/gatsby-core-utils/src/__tests__/create-content-digest.ts#L10).

The benefit is that the `expect(sharp).toHaveBeenCalledTimes(n)` assertions match reality now: have updated those throughout the file. Now the existing assertions properly cover the favicon generation, and I've added a new test case to show the favicon is skipped when `include_favicon` is false.

Reply:
Thanks for fixing this.